src/main/java/com/northconcepts/datapipeline/examples/cookbook/OracleInsertMultiRowsUsingSelectUnionAll.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2006-2023 North Concepts Inc.  All rights reserved.
+ * Proprietary and Confidential. Use is subject to license terms.
+ *
+ * https://northconcepts.com/data-pipeline/licensing/
+ */
+package com.northconcepts.datapipeline.examples.cookbook;
+
+import com.northconcepts.datapipeline.core.DataException;
+import com.northconcepts.datapipeline.core.DataReader;
+import com.northconcepts.datapipeline.core.DataWriter;
+import com.northconcepts.datapipeline.csv.CSVReader;
+import com.northconcepts.datapipeline.csv.CSVWriter;
+import com.northconcepts.datapipeline.jdbc.JdbcReader;
+import com.northconcepts.datapipeline.jdbc.JdbcWriter;
+import com.northconcepts.datapipeline.jdbc.insert.OracleMultiRowSelectUnionAllStatementInsert;
+import com.northconcepts.datapipeline.job.Job;
+
+import java.io.File;
+import java.io.OutputStreamWriter;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+
+public class OracleInsertMultiRowsUsingSelectUnionAll {
```

Comment:
Please rename this class to `InsertMultipleRowsToOracleUsingSelectUnion` to match the content's title.